newrelic/build.go

```diff
@@ -85,6 +85,18 @@ func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 		}
 	}
 
+	if _, ok, err := pr.Resolve("new-relic-python"); err != nil {
+		return libcnb.BuildResult{}, fmt.Errorf("unable to resolve new-relic-python plan entry\n%w", err)
+	} else if ok {
+		dep, err := dr.Resolve("new-relic-python", "")
+		if err != nil {
+			return libcnb.BuildResult{}, fmt.Errorf("unable to find dependency\n%w", err)
+		}
```

Comment:
Again, unless you want the buildpack to download/extract/install something then you don't need to do any dependency resolution. You can simply call the tool you use to install the package, i.e. pip.

```suggestion
```

Reply:
Done